ftl_installer/parser/Parser.py

```diff
@@ -2,6 +2,17 @@
 import argparse
 import ftl_installer
 import logging
+import pprint
+import yaml
+
+class ConfigHandler(object):
```

Comment:
If the ConfigHandler class was going to remain small and only be used within the Parser module, then embedding it here would be fine.

In this case, the class that represents the config should be in it's own module (ftl_installer.config.Config.py possibly). This will make it easier toadd and manage the full set of actions that will be needed eventually.

I would also suggest just calling the class Config instead of ConfigHandler.